### PR TITLE
Add where parameter for filtering productTypes

### LINKED ISSUE #11: Add a possibility to select only productTypes which should be exported
### Description
Based on this issue https://github.com/commercetools/express-impex/issues/156 we should add an input parameter with a comma separated list of productType keys which should be exported.

### PR DESCRIPTION
**Pull request by junajan**
Fixes #11 

- Tests
    - [x] Unit
    - [ ] Integration
    - [ ] Acceptance
- [x] Documentation

### CODE REVIEW

File: tests/unit/product-type-export.spec.js [hunk 1]
```diff
@@ -103,11 +104,53 @@ test(`productType import module
...
 })
+
+test(`productType export module
+  should filter productTypes if where parameter is present`, (t) => {
+  const exporter = new ProductTypeExport(_.cloneDeep(options))
+  const expectedFilter = 'key IN ("a", "b")'
+  const mockClient = {
+    condition: '',
+    where (params) {
+      this.condition = params
```
Comment (hisabimbola) [inline tests/unit/product-type-export.spec.js]: Hmmmm, very tricky and buggy I must say _smart one_ 😀 

but good solution 😉

File: src/product-type-export.js [hunk 2]
```diff
@@ -246,11 +247,22 @@ export default class ProductTypeImport {
...
+    return clientEndpoint
+  }
+
   downloadProductTypes (file) {
     const writeStream = createWriteStream(file)
     writeStream.write('[')
     let isFirst = true
-    return this.client.productTypes.process(({
+
+    return this.getProductTypeClient()
```
Comment (Siilwyn) [inline src/product-type-export.js]: The indentation seems wrong here?

File: src/product-type-export.js [hunk 3]
```diff
@@ -246,11 +247,22 @@ export default class ProductTypeImport {
...
+    .process(({
```
Comment (Siilwyn) [inline src/product-type-export.js]: The indentation seems wrong here? Would expect it to be more to the right.

Comment (junajan) [inline src/product-type-export.js]: Hmm.. there are other places indented like this, for example:
https://github.com/sphereio/sphere-product-type-export/pull/15/files/2f4e1dc682a548d14677d9fe1eed22d1c843e561#diff-9d42f85cc847a16773c7454182538dc7R581

Comment (Siilwyn) [inline src/product-type-export.js]: Yeah I guess it's also personal preference. I think indenting those chains makes it a more readable as you can easily see where the chain comes from.

**Pull request merged**